en/components/chip.md

```diff
@@ -0,0 +1,162 @@
+---
+
+### Chip
+
+`igxChip` Chips are compact components that represent discrete information.
+
+#### Demo
+
+<div class="sample-container loading" style="height:550px">
+    <iframe id="chip-iframe" src='{environment:demosBaseUrl}/chip-sample' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<br/>
+<div>
+<button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="chip-sample-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+
+#### Initializing Chips
+
+The `IgxChipComponent` is the main class for a chip elemenent and the `IgxChipsAreaComponent` is the main class for the chip area, it is used to manage the chips defined inside it. It is used when handling more complex scenarios that require interaction between chips (dragging, selection, navigation and etc.).The IgxChipComponent requires an id to be defined so that the different chips can be easily distinguished.
+
+Example of using `igxChip` with `igxChipArea`:
+
+```html
+<igx-chips-area>
+    <igx-chip *ngFor="let chip of chipList" [id]="chip.id">
+        <label igxLabel>{{chip.text}}</label>
+    </igx-chip>
+</igx-chips-area>
+```
+
+### Features
+
+#### Selection
+
+Selection is disabled by default, but can be enabled with an option called `selectable`. The selecting is done by focusing the chip either by clicking of using the `Tab` button and the pressing `Space`. An event is fired when the selection state of the `igxChip` changes. If a chip is already selected it can be deselected by pressing the `Space` button again while the chip is focused.
+
+```html
+<igx-chips-area>
+    <igx-chip *ngFor="let chip of chipList" [selectable]="'true'">
+        <label igxLabel>{{chip.text}}</label>
+    </igx-chip>
+</igx-chips-area>
+```
+
+```typescript
+public ngOnInit() {
+    chip-area.forEach((chip) => {
+        column.selectable = true;
+    });
+}
+```
+
+#### Removing
+
+The `remove button` is part of the chip area as well. Developers can allow end-users to remove chips. You can handle the removing of the chips template where the chips are defined. This can be done by binding to the event `onRemove` that each chip provides.
+
+```html
+<igx-chips-area>
+    <igx-chip *ngFor="let chip of chipList" [removable]="'true'">
+        <label igxLabel>{{chip.text}}</label>
+    </igx-chip>
+</igx-chips-area>
+```
+
+```typescript
+public ngOnInit() {
+    chip-area.forEach((chip) => {
+        column.removable = true;
+    });
+}
+```
+
+#### Moving/Dragging
+
+The chip can be dragged in order to change it's position. The moving/dragging is disabled by default. but can be enabled with an option called `draggable`. The user need to handle the actual moving of the chip inside their template themselves.
```

Comment:
There should be comma instead of dot after `The moving/dragging is disabled by default`